Jenkinsfile

```diff
@@ -870,22 +870,30 @@ pass = ${OBS_CREDENTIALS_PASSWORD}
             }
             // Save logs of failed builds to s3 - we want to analyze where we may have jenkins issues.
             script {
-                if ((env.BRANCH_NAME == 'develop' || env.BRANCH_NAME == 'master') && fileExists('bin/clean-jenkins-log')) {
+                if (env.BRANCH_NAME == 'develop' || env.BRANCH_NAME == 'master') {
                     writeFile(file: 'build.log', text: getBuildLog())
-                    sh 'bin/clean-jenkins-log'
+                    sh """ 
```

Comment:
I think these should be 2 sh calls:
```
sh 'bin/clean-jenkins-log'
sh 'container-host-files/opt/scf/bin/klog.sh .......'
```

Reply:
Why ?

Reply:
Clearer visualization on Jenkins.

Reply:
Better separation in the log output then ?

Reply:
IMO yes.

Reply:
Ok. Then agreed, two calls it should be

Reply:
still not convinced how two subsequent calls will provide better logging. To my understanding, since both scripts intend log collection therefore seems logical to group them together.

Reply:
https://jenkins-new.howdoi.website/blue/rest/organizations/jenkins/pipelines/scf/branches/PR-2170/runs/6/nodes/81/steps/125/log/?start=0

They are grouped.

Reply:
After looking at the logs, splitting it into two separate calls.